### DISCUSSION TITLE
can't update

### DISCUSSION THREAD

**Discussion by a-random-guy-with-2-brain-cells** [bridge-core/editor#1182] [Q&A] [answered]
how to install older version of bridge
because my version is on 2.7.22 and i cant update it 
the installer say that he c'ant update because i d'ont have the .msi file

**Comment by outercloudstudio** [comment, marked as answer]
You can download specific versions directly from the releases pages. https://github.com/bridge-core/editor/releases

**Reply by a-random-guy-with-2-brain-cells** [reply]
ho okay
tank you :)
